package.json

```diff
@@ -2,7 +2,7 @@
   "name": "panzoom",
   "version": "2.5.0",
   "description": "Pan and zoom SVG elements",
-  "main": "index.js",
+  "main": "dist/panzoom.min.js",
```

Comment:
minifies the code into to make it more efficient when rand on a website...Essentially compresses it

Why You Should Compress JavaScript
There are a number of reasons why compressing your JavaScript files is a good idea:
Quicker download times for your users.
Reduced bandwidth consumption of your website.
Reduced number of HTTP requests on your server when combining many JavaScript files into one compressed file, thus reducing the server load and allowing more visitors to access your website.
Comments and whitespace are not needed for JavaScript execution; Removing them will reduce file size and speed up script execution times.